.gitignore

```diff
@@ -30,3 +30,6 @@ phpcs.xml
 phpunit.xml
 phpunit-watcher.yml
 .tool-versions
+
+# Config file for VSCode debugger
+.vscode/launch.json
```

Comment:
Let’s avoid listing files related to IDE that folks can ignore globally on their machines. We probably should explain that in the docs based on https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#create-a-global-gitignore.

Reply:
Alright I removed this `.gitignore` addition, tested that adding to a global gitignore really works, and documented it in the `wp-env` README. Now the patch only adds stuff to documentation and doesn't do anything else.